s64da_benchmark_toolkit/prepare.py

```diff
@@ -266,23 +266,24 @@ def add_indexes(self):
                 print(f'Applying {sql_file_path}')
                 with open(sql_file_path, 'r') as sql_file:
                     sql = sql_file.read()
-                    if sql:
-                        conn.cursor.execute(sql)
+
+                tasks = [self.psql_exec_cmd(cmd) for cmd in sql.split(';')]
+                self._run_tasks_parallel(tasks)
 
     def vacuum_analyze(self):
         print(f'Running VACUUM on {self.args.dsn}')
-        self._run_shell_task(f'psql {self.args.dsn} -c "VACUUM"')
+        self._run_shell_task(self.psql_exec_cmd('VACUUM'))
```

Comment:
for the columnstore it would be good to run all the table vacuum commands independently and in parallel

Reply:
Yeah, I am thinking about #19 and that will fix it then.